### PR TITLE
Fix create api, broken after upgrading request package

### PR DESCRIPTION
**Pull request by akanduru**
As per request@2.83.0 documentation,  changed args.body to JSON object.

### CODE REVIEW

File: src/adapters/node.js [hunk 1]
```diff
@@ -8,7 +8,10 @@
         defer: Q.defer,
         http: function(args) {
             var deff = Q.defer();
-            args.body = args.data;
+            if(args.data && typeof args.data === "string") {
+              // Setting args.json requires args.body to be a JSON as per request docs.
+              args.body = JSON.parse(args.data);
```
Comment (plynchnlm) [inline src/adapters/node.js:12]: Could use "else args.body=args.data"

File: src/adapters/node.js [hunk 2]
```diff
@@ -8,7 +8,10 @@
         defer: Q.defer,
```
Comment (plynchnlm) [inline src/adapters/node.js:7]: The comment at the top of the file says that this file was generated from CoffeeScript, which is no longer true (or anyway we could not find the CoffeeScript source).  I suggest you remove the comment.

**Review by plynchnlm — CHANGES_REQUESTED** [review]

File: src/adapters/node.js (hunk 1 above)
Comment (akanduru) [reply src/adapters/node.js:12]: done.

File: src/adapters/node.js (hunk 2 above)
Comment (akanduru) [reply src/adapters/node.js:7]: Removed.

**Review by plynchnlm — APPROVED** [review]

**Pull request merged**